commec/screen.py

```diff
@@ -265,7 +265,7 @@ def screen_nucleotides(self):
         logging.debug("\t...checking blastn results")
         check_for_regulated_pathogens(self.databases.nucleotide_db.out_file,
                                        self.params.db_dir,
-                                       str(self.params.inputs.threads))
+                                       str(self.params.config.threads))
 
     def screen_benign(self):
```

Comment:
While we're editing this, should we move this check to the beginning of the method?

```
        if not os.path.exists(sample_name + ".reg_path_coords.csv"):
            logging.info("\t...no regulated regions to clear\n")
return
```

This is another place where we're using "this file exists" as a proxy for "regulated regions were found", and I think in the (future) JSON output refactor we're going to want to actually check the outcome of that step. Another question we'd want to easily be able to ask the JSON structure ― were any regulated regions identified?